templates/openstack/openstack-cheat-sheet.html

```diff
@@ -0,0 +1,38 @@
+{% extends "openstack/_base_openstack.html" %}
+
+{% block title %}The most comprehensive OpenStack cheat sheet{% endblock %}
+{% block meta_description %}Instances (Nova), networks (Neutron), images (Glance), volumes (Cinder), identities (Keystone) and much more. Get started with OpenStack quickly.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1q37DrtjZhcgRK4NuL3CUsCv1aU6AQDX5uqtQ4ndVT-A/edit{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru-topped">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h1>OpenStack Cheat Sheet</h1>
+      <p>The most comprehensive OpenStack cheat sheet for beginners and more advanced users.</p>
+      <p> What’s included:</p>
+        <ul class="p-list">
+          <li class="p-list__item is-ticked"> Keystone </li>
+          <li class="p-list__item is-ticked"> Glance </li>
+          <li class="p-list__item is-ticked"> Neutron </li>
+          <li class="p-list__item is-ticked"> Nova </li>
+          <li class="p-list__item is-ticked"> Cinder </li>
+        </ul>
+      <p>Get the following OpenStack cheat sheet for free. No registration needed!</p>
+      <a href="https://assets.ubuntu.com/v1/8d3130a1-OpenStack.cheat.sheet.1.pdf?_ga=2.38929315.1481755299.1641543595-1851044449.1622191471" class="p-button--positive">Get OpenStack cheat sheet</a>
```

Comment:
This is not good English.
```suggestion
      <a href="https://assets.ubuntu.com/v1/8d3130a1-OpenStack.cheat.sheet.1.pdf?_ga=2.38929315.1481755299.1641543595-1851044449.1622191471" class="p-button--positive">Get the OpenStack cheat sheet</a>
```